kai/llm_io_handler.py

```diff
@@ -70,6 +70,7 @@ def get_key_and_res_function(
 async def get_incident_solutions_for_file(
     model_provider: ModelProvider,
     incident_store: IncidentStore,
+    jinja_env: Environment,
```

Comment:
I'd remove this as an argument and just have llm_io_handler build out the template. That way if we ever change the format we don't need to worry about plumbing it through as much